### ISSUE TITLE
PyTorch Lightning example is broken

### ISSUE THREAD

**Issue by charlesfrye** [wandb/examples#66]
The input validation complains because we're using the logits with accuracy, which I swear used to work.

**Comment by charlesfrye** [comment]
(work as in be calculated correctly, not just run without complaining)

**Comment by charlesfrye** [comment]
fixed by d82e9f0
